### PR TITLE
Making SDK attributes match agent

### PR DESCRIPTION
**Pull request by RyanKadri**
This PR adds alias names for attributes where the Python agent disagrees with this instrumentation library. After this is merged and has some time to bake in Grok, we should migrate the UI to use the new field names (from the python agent) and then remove the old attribute names.

### CODE REVIEW

File: src/nr_openai_observability/bedrock.py [hunk 1]
```diff
@@ -321,25 +321,45 @@ def build_bedrock_events(response, event_dict, time_delta):
...
-            summary["usage.prompt_tokens"] = input_tokens
+            summary.update(
+                compat_fields(
+                    ["usage.prompt_tokens", "response.usage.prompt_tokens"],
+                    input_tokens,
+                )
+            )
         if max_tokens:
-            summary["max_tokens"] = max_tokens
+            summary.update(["max_tokens", "request.max_tokens"], max_tokens)
```
Comment (bonczj) [inline src/nr_openai_observability/bedrock.py:362]: I think this one is missing the `compat_fields` function wrapper.

File: src/nr_openai_observability/build_events.py [hunk 2]
```diff
@@ -193,19 +266,35 @@ def build_embedding_event(response, request, response_headers, response_time):
...
+            ["usage.prompt_tokens", "response.usage.prompt_tokens"],
+            response.usage.prompt_tokens,
+        ),
+        **compat_fields(
+            ["response.api_version", "response.headers.llmVersion"],
+            response_headers.get("openai-version"),
+        ),
+        **compat_fields(["api_type", "response.api_type"], response.api_type),
+        **compat_fields(
+            ["api_version", "response.api_version"],
```
Comment (bonczj) [inline src/nr_openai_observability/build_events.py:292]: Do we need a second copy of `response_headers.get("openai-version")` under new names?

**Review by bonczj — COMMENTED** [review]
Just a few small questions.

File: src/nr_openai_observability/bedrock.py (hunk 1 above)
Comment (RyanKadri) [reply src/nr_openai_observability/bedrock.py:362]: Good catch!

File: src/nr_openai_observability/build_events.py (hunk 2 above)
Comment (RyanKadri) [reply src/nr_openai_observability/build_events.py]: Nope. Good call. Looks like we ended up with a duplicate

**Review by bonczj — APPROVED** [review]
:shipit:

**Review by bonczj — APPROVED** [review]
🚢

**Pull request merged**